src/test/java/net/snowflake/client/jdbc/FileUploaderPrep.java

```diff
@@ -241,11 +241,111 @@ abstract class FileUploaderPrep extends BaseJDBCTest {
           + "  \"message\": null,\n"
           + "  \"success\": true\n"
           + "}";
+  private final String exampleGCSJsonStringWithUseRegionalUrl =
+      "{\n"
```

Comment:
How ugly it is... It could be loaded from a file :)

Reply:
could be, I will change it